docs/index.rst

```diff
@@ -22,7 +22,7 @@
     : --
 
 .. Procrustes documentation master file, created by
-   sphinx-quickstart on Wed Apr 11 19:35:53 2018.
+   sphinx-quickstart on Wed Apr 11 19:35:53 2020.
```

Comment:
This `2018` should not also be changed to `2020`.

Reply:
Fixed in abc0c35.